WalletWasabi/WabiSabi/Backend/DoSPrevention/Warden.cs

```diff
@@ -62,6 +62,11 @@ private static Prison DeserializePrison(string prisonFilePath)
 			Logger.LogInfo($"{banned} banned UTXOs are found in prison.");
 		}
 
+		if (longBanned > 0)
+		{
+			Logger.LogInfo($"{longBanned} force-banned UTXOs are found in prison.");
```

Comment:
I think `long-banned` is better.
```suggestion
			Logger.LogInfo($"{longBanned} long-banned UTXOs are found in prison.");
```